Develop to master (#6)
golemfactory/yagna-docs: provider-tutorials/provider-tutorial.md

#### Platforms

> This is our very second Alpha Provider Sneak Peek reveal. Therefore, temporarily, Golem official support is for:

Author: ```suggestion
This is still an early Alpha Provider Sneak Peek reveal. Therefore, temporarily, Golem official support is for:
```

│  Wallet                                                      │
│                                                              │
│  address         0xe4781cd3af959417f560372544f77aec33124b5f  │
│  amount (total)  0 GLM                                       │
│      (on-chain)  0 GLM                                       │
│       (zk-sync)  0 GLM                                       │
│                                                              │
│  pending         0 GLM (0)                                   │
│  issued          0 GLM (0)                                   │
├──────────────────────────────────────────────────────────────┤
│  Tasks                                                       │
│                                                              │
│  last 1h processed    0                                      │
│  last 1h in progress  0                                      │
│  total processed      0                                      │
└──────────────────────────────────────────────────────────────┘

```

{% hint style="info" %}
Under your address you can see both **on-chain** and **zk-sync** values listed. 

> Although zk-synk is from now on the main payment operator in Golem you may receive on-chain transactions as well. To confirm correctness of the listed values head over to [https://rinkeby.etherscan.io/](https://rinkeby.etherscan.io/) \(on-chain\) and [https://rinkeby.zkscan.io/](https://rinkeby.zkscan.io/) \(for zk-synk\).

Author: ```suggestion
Although zk-sync is from now on the default payment driver in Golem you may receive on-chain transactions as well. To confirm correctness of the listed values head over to [https://rinkeby.etherscan.io/](https://rinkeby.etherscan.io/) \(on-chain\) and [https://rinkeby.zkscan.io/](https://rinkeby.zkscan.io/) \(for zk-sync\).
```
